test/playwright/utils/navigation.ts

```diff
@@ -1,10 +1,41 @@
 import { expect, Page } from '@playwright/test'
 
+import {
+  ALL_MEDIA,
+  AUDIO,
+  IMAGE,
+  MediaType,
+  MODEL_3D,
+  SearchType,
+  VIDEO,
+} from '~/constants/media'
+import messages from '~/locales/en.json'
+
 const buttonSelectors = {
   filter: '[aria-controls="filters"]',
   contentSwitcher: '[aria-controls="content-switcher-modal"]',
 }
 
+export function sleep(ms: number) {
+  return new Promise<void>((resolve) =>
+    setTimeout(async () => {
+      resolve()
+    }, ms)
```

Comment:
Totally a nit, but this can be cut down (and `async` declaration isn't necessary either way here):

```suggestion
    setTimeout(resolve, ms)
```